test/kwtool_web/plugs/check_empty_body_params_plug_test.exs

```diff
@@ -35,6 +36,7 @@ defmodule KwtoolWeb.CheckEmptyBodyParamsPlugTest do
       conn =
         conn
         |> Map.put(:body_params, %{name: "John Doe"})
+        |> Map.put(:method, "POST")
         |> CheckEmptyBodyParamsPlug.call([])
 
       assert conn.halted == false
```

Comment:
Should we add a test for get/delete params without body that it should not be halted. 💡